www/c-icap/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		c-icap
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		c-icap connects your Proxy with a virus scanner
+PLUGIN_DEPENDS=		c-icap c-icap-modules clamav
```

Comment:
c-icap-modules depends on clamav, yes. But isn't it clearer if it's listed here? Or would this break something?